README.md

```diff
@@ -1,15 +1,17 @@
 # Turso CLI
 
-[![goreleaser](https://github.com/tursodatabase/turso-cli/actions/workflows/goreleaser.yml/badge.svg)](https://github.com/tursodatabase/turso-cli/actions/workflows/goreleaser.yml)
+[![Go](https://github.com/tursodatabase/turso-cli/actions/workflows/go.yml/badge.svg)](https://github.com/tursodatabase/turso-cli/actions/workflows/go.yml)
```

Comment:
Why are you removing goreleaser? I think both are useful.

Reply:
Originally we just used `go.yml`. 

But [it always failed ](https://github.com/tursodatabase/turso-cli/actions/runs/6461516569) because of conflicts when we restored the cache.
<img width="1344" alt="Screenshot 2023-10-10 at 8 26 05 AM" src="https://github.com/tursodatabase/turso-cli/assets/7853668/4ba98b03-4392-4562-97f2-fb4ae5c9e624">
This resulted in the errors you see above and also on a red status badge.
That is why @gris changed it to `goreleaser.yml` a few days ago.

I think a CI badge makes more sense. Meaning that the "last CI run was successful for this project".
I don't see much value in a badge saying "the last release was successful". But I also don't oppose having it.
Let me know if you still want to keep `gorereleaser` badge and I'll add it back. @gris let me know what you think as well.

Reply:
Makes a lot of sense! Thanks for spotting that